File: LeetCode/Greedy/Python/Leetcode1217-minimum-cost-to-move-chips-to-the-same-position.py
class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        o,e = 0,0
        for p in position:
            if p % 2: o += 1
            else: e += 1
        if not e or not o: return 0
        return min(o,e)

File: LeetCode/Greedy/Rust/Leetcode1217-minimum-cost-to-move-chips-to-the-same-position.rs
impl Solution {
    pub fn min_cost_to_move_chips(position: Vec<i32>) -> i32 {
        let (mut o, mut e) = (0, 0);
        for p in &position {
            if p % 2 == 0 {
                o += 1;
            } else {
                e += 1;
            }
        }
        if o == 0 || e == 0 {
            return 0;
        }
        o.min(e)
    }
}
